www/images/icon-external.svg

```diff
@@ -1 +1 @@
-<svg width="15" height="15" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M7 3.5a.5.5 0 000-1v1zM12.5 8a.5.5 0 00-1 0h1zM7 2.5H3v1h4v-1zM.5 5v7h1V5h-1zM3 14.5h7v-1H3v1zm9.5-2.5V8h-1v4h1zM10 14.5a2.5 2.5 0 002.5-2.5h-1a1.5 1.5 0 01-1.5 1.5v1zM.5 12A2.5 2.5 0 003 14.5v-1A1.5 1.5 0 011.5 12h-1zM3 2.5A2.5 2.5 0 00.5 5h1A1.5 1.5 0 013 3.5v-1zM9 .5a.5.5 0 000 1v-1zm5 .5h.5a.5.5 0 00-.5-.5V1zm-.5 5a.5.5 0 001 0h-1zM9 1.5h5v-1H9v1zm4.5-.5v5h1V1h-1z" fill="#999" fill-rule="nonzero"/><path shape-rendering="crispEdges" stroke="#999" stroke-linecap="round" d="M14 1L9 6"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="15" fill="none" stroke="#999" stroke-linecap="round" stroke-linejoin="round"><path d="M7 3H3a2 2 0 0 0-2 2m0 0v7a2 2 0 0 0 2 2h7a2 2 0 0 0 2-2V8M9 1h5v5m0-5L9 6" shape-rendering="crispEdges"/></svg>
```

Comment:
The rich diff on this one is a bit off

Reply:
Ah, yep, good catch. Will fix

Reply:
Should look the same now — the `crispEdges` was non-deterministically snapping the corner depending on various viewing factors.